eng/pipelines/publish-dev-release.yml

```diff
@@ -0,0 +1,90 @@
+trigger: none
```

Comment:
How do we trigger this manually?

Reply:
It will be through the Azure DevOps page, I'll send a link once I have the pipeline created.